### PR TITLE
feat: add few faqs

### PR DESCRIPTION
**Pull request by jainpawan21**
### What change does this PR introduce?

Added few FAQs and missing info

### CODE REVIEW

File: docs/docs/platform/subscribers.md [hunk 1]
```diff
@@ -83,3 +83,15 @@ await novu.subscribers.remove(user.id);
...
+## Frequently Asked Questions
+
+<details>
+  <summary>How to store custom properties in subscriber</summary>
+  <p>Subscribers have fixed schema. Storing custom properties is not supported but here is a work around, you can store that property in your database and send those values in payload option of trigger using variables. Read more about variables <a href="./templates#variable-usage"> here </a>.</p>
+</details>
+
+<details>
+  <summary>How to get subscriber properties before step execution in workflow</summary>
+  <p>Workflow has access to all existing properties of subscriber as well as payload variables. So there is no any extra step needed</p>
```
Comment (ainouzgali) [inline docs/docs/platform/subscribers.md:96]:
```suggestion
  <p>Workflow has access to all existing properties of subscriber as well as payload variables. So no extra steps are needed</p>
```

**Review by ainouzgali — COMMENTED** [review]
Hi @jainpawan21 , let me know what you think of my suggestions :)

**Review by p-fernandez — APPROVED** [review]
Adding @ainouzgali suggestions and is done.
🌟

**Comment by jainpawan21** [comment]
@ainouzgali added your suggestions ✅

**Pull request merged**